conda/base/context.py

```diff
@@ -821,6 +821,8 @@ def user_agent(self):
         builder.append("%s/%s" % self.os_distribution_name_version)
         if self.libc_family_version[0]:
             builder.append("%s/%s" % self.libc_family_version)
+        if self.experimental_solver.value != "classic":
+            builder.append("solver/%s" % self.experimental_solver.value)
```

Comment:
Interesting that it's so easy! Do you think we can try adding the version of it as well?

Reply:
Mmm, that requires running the `_get_solver_class()` function, importing the class, and expecting a version field somewhere. We haven't standardized this, and I was trying to keep this as agnostic as possible. If we want more details, we might want to rethink this; e.g. offering an API for downstream plugins to add stuff to the user agent string.

The alternative right now:

* Implement this from the `conda-libmamba-solver` side without such an API and revisit in the future.
* Quick API: add a `user_agent` property to the Solver class, and have the plugin define it?

There are lots of ways of fixing this but once we choose one we need to stick to it, I guess, so better think it through.